debugging/README.md

```diff
@@ -0,0 +1,29 @@
+## Debugging API
+- [Chrome Debugging Protocol](https://chromedevtools.github.io/debugger-protocol-viewer/)
+- [V8 Debugging Protocol](https://github.com/v8/v8/wiki/Debugging-Protocol)
+
+Node currently relies on V8's internal Debug API and associated commands and events. The Debug API is published at [v8/include/v8-debug.h](https://gihtub.com/v8/v8/blob/master/include/v8-debug.h), and the message protocol is documented in prose in [the V8 wiki](https://github.com/v8/v8/wiki/Debugging-Protocol) and in code in [v8/src/debug/debug.js#L2341-2372](https://github.com/v8/v8/blob/master/src/debug/debug.js#L2341-L2372). 
```

Comment:
Typo: s/gihtub/github/

Note that the links to v8/v8/blob/master will go stale pretty quickly.  Consider replacing master with a release tag.